src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,20 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  width: $catalog-width;
+  padding: $padding-big 0;
+
+  &--cards {
+    margin: 0 auto;
+  }
+
```

Comment:
Strange solution, don't use modifier for margins, use mix of block and element for this purpose